orc8r/tools/fab/python_utils.py

```diff
@@ -0,0 +1,35 @@
+"""
+Copyright 2022 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+
+def strtobool(boolstring):
```

Comment:
LGTM with small nit: 
type-hints on new python code would be nice :)
```suggestion
def strtobool(boolstring: str) -> bool:
```

Reply:
Fixed.